CalPatRec/src/AgnosticHelixFinder_module.cc

```diff
@@ -785,8 +789,8 @@ namespace mu2e {
 
     // order from largest z to smallest z (skip over stopping target and calo cluster since they
     // aren't in _chColl)
-    std::sort(_tcHits.begin() + sortStartIndex, _tcHits.end(), [&](const cHit& a, const cHit& b) {
-        return _chColl->at(a.hitIndice).pos().z() > _chColl->at(b.hitIndice).pos().z();
+    std::sort(_tcHits.begin(), _tcHits.end(), [&](const cHit& a, const cHit& b) {
```

Comment:
Since z doesn't change since construction, it would be more efficient to use std::set and sort on insertion.

Reply:
Thanks for the comments. @matthewstortini will make another round of optimization in the future, but now it's out of the scope of this PR.